src/EPPlus/ConditionalFormatting/ExcelConditionalFormattingCollection.cs

```diff
@@ -1,1005 +1,1302 @@
-/*************************************************************************************************
-  Required Notice: Copyright (C) EPPlus Software AB. 
-  This software is licensed under PolyForm Noncommercial License 1.0.0 
-  and may only be used for noncommercial purposes 
-  https://polyformproject.org/licenses/noncommercial/1.0.0/
-
-  A commercial license to use this software can be purchased at https://epplussoftware.com
- *************************************************************************************************
-  Date               Author                       Change
- *************************************************************************************************
-  01/27/2020         EPPlus Software AB       Initial release EPPlus 5
- *************************************************************************************************/
+﻿using OfficeOpenXml.ConditionalFormatting.Contracts;
```

Comment:
Looks like the header has been removed here and in some other files.

Reply:
I've added it back in to all files and to new ones now